Marlin/src/lcd/menu/menu_info.cpp

```diff
@@ -49,26 +49,29 @@
     );
 
     char buffer[21];
+
     printStatistics stats = print_job_timer.getStats();
 
+    #define STATIC_PAIR(MSG, VALUE, CNTR)    do {strcpy_P(buffer, PSTR(": ")); strcpy(buffer, VALUE); STATIC_ITEM(MSG, CNTR, false, buffer);} while(0)
+    #define STATIC_PAIR_P(MSG, PVALUE, CNTR) do {strcpy_P(buffer, PSTR(": ")); strcpy_P(buffer, PSTR(PVALUE)); STATIC_ITEM(MSG, CNTR, false, buffer);} while(0)
+
     START_SCREEN();                                                                                // 12345678901234567890
-    STATIC_ITEM(MSG_INFO_PRINT_COUNT ": ", false, false, i16tostr3left(stats.totalPrints));        // Print Count: 999
-    STATIC_ITEM(MSG_INFO_COMPLETED_PRINTS": ", false, false, i16tostr3left(stats.finishedPrints)); // Completed  : 666
+    STATIC_PAIR(MSG_INFO_PRINT_COUNT, i16tostr3left(stats.totalPrints), false, false);             // Print Count: 999
+    STATIC_PAIR(MSG_INFO_COMPLETED_PRINTS, i16tostr3left(stats.finishedPrints), false, false);     // Completed  : 666
```

Comment:
These only take 3 arguments. Did you intend for there to be both a `CNTR` and an `NVRT` flags?